oauth2-authorization-server/src/main/java/org/springframework/security/oauth2/server/authorization/oidc/web/OidcUserInfoEndpointFilter.java

```diff
@@ -66,6 +69,9 @@ public final class OidcUserInfoEndpointFilter extends OncePerRequestFilter {
 	private final HttpMessageConverter<OAuth2Error> errorHttpResponseConverter =
 			new OAuth2ErrorHttpMessageConverter();
 
```

Comment:
Please add `private AuthenticationConverter authenticationConverter;`

Reply:
Currently the Authentication is resolved from the `SecurityContext`, not from the `HttpServletRequest` - hence my omitting the `AuthenticationConverter`. If we do want to support authentication converters for consistency, the default converter (created in the Configurer) should be:

```java
AuthenticationConverter authenticationConverter = request -> {
		Authentication principal = SecurityContextHolder.getContext().getAuthentication();
		return new OidcUserInfoAuthenticationToken(principal);
	};
```

Does that make sense?